packages/yourloops/lib/auth/hook.tsx

```diff
@@ -506,6 +507,16 @@ export function AuthContextImpl(api: AuthAPI): AuthContext {
     // return unmount;
   };
 
+  const certifyProfessionalAccount = async (): Promise<void> => {
+    const { frProId } = await api.certifyProfessionalAccount();
```

Comment:
It is necessary to do this if user is null?
If not, I would move it into the if condition :slightly_smiling_face: